### PR TITLE
11139 f default lookup

### PR DESCRIPTION
**Pull request by petrican**
https://www.loom.com/share/b92b964e361a4d69ab48ad1237d03459

Consider default values for Lookup, Datetimerange and Amount. If others will be needed they will be added in a sep issue.

### CODE REVIEW

File: frontend/src/components/widget/DatetimeRange.js [hunk 1]
```diff
@@ -205,6 +225,11 @@ DatetimeRange.propTypes = {
   mandatory: PropTypes.any,
   validStatus: PropTypes.any,
   timePicker: PropTypes.any,
+  isFilterActive: PropTypes.bool,
+  defaultValue: PropTypes.string,
```
Comment (siemiatj) [inline frontend/src/components/widget/DatetimeRange.js:229]: You can remove `value/valueTo` from props

File: frontend/src/components/widget/WidgetRenderer.js [hunk 2]
```diff
@@ -172,7 +178,13 @@ class WidgetRenderer extends PureComponent {
       tabIndex,
       onShow,
       onHide,
+      isFilterActive: filterActiveState,
+      defaultValue: widgetData[0].defaultValue,
```
Comment (siemiatj) [inline frontend/src/components/widget/WidgetRenderer.js:182]: I assume we don't want to use `value` as a backup here, right ?

**Review by siemiatj — APPROVED** [review]
Nothing major, good to merge

File: frontend/src/components/widget/DatetimeRange.js (hunk 1 above)
Comment (petrican) [reply frontend/src/components/widget/DatetimeRange.js:229]: not quite we use defaultValue, defaultVauleTo as default ones and value,/valueTo as the normal ones

**Comment by petrican** [comment]
Will do some more tests on the dev instance and then I will merge. Thanks @kuba

**Pull request merged**